core-plugins/src/test/java/io/cdap/plugin/batch/joiner/JoinerConfigTest.java

```diff
@@ -59,6 +63,12 @@
   private static final String selectedFields = "film.film_id, film.film_name, " +
     "filmActor.actor_name as renamed_actor, filmCategory.category_name as renamed_category";
 
+  private static final String stage = "stage";
+  private static final String mockStage = "mockstage";
+  private static final String SELECTED_FIELDS = "selectedFields";
```

Comment:
test code should reference the constants in the config class instead of re-defining them here

Reply:
Fixed